target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8074-x80-5g.dts

```diff
@@ -0,0 +1,476 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2022, Robert Marko <robimarko@gmail.com> */
+/* altered by Jonathan Brophy <professor_jonny@hotmail.com> */
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
+#include "ipq8074-ess.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "tplink x80-5g";
+	compatible = "tplink,x80-5g", "qcom,ipq8074";
+
+	aliases {
+		 //Aliases as required by u-boot to patch MAC addresses
+		 serial0 = &blsp1_uart5;
+ 		 ethernet0 = &dp5; // Aliases to capture wan mac from bootloader
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " root=/dev/ubiblock0_1";
+	};
+
+	modem {
+		compatible = "gpio-export";
+		pinctrl-0 = <&modem_pins>;
+		pinctrl-names = "default";
+
+		w_disable {
+			gpio-export,name = "w_disable";
+			gpio-export,output = <1>;
+			gpios = <&tlmm 35 GPIO_ACTIVE_HIGH>; // wwan disable output
+		};
+
+		modem-reset {
+			gpio-export,name = "modem_reset";
+			gpio-export,output = <0>;
+			gpios = <&tlmm 29 GPIO_ACTIVE_HIGH>; // reset modem output
+		};
+	};
+
+		antennas {
+		compatible = "gpio-export";
+		pinctrl-0 = <&antenna_pins>;
+		pinctrl-names = "default";
+
+		ext_antenna_en {
+			gpio-export,name = "ext_antenna_en";
+			gpio-export,output = <0>;
+			gpios = <&tlmm 56 GPIO_ACTIVE_HIGH>; // select external sma antennas for WWan output
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		pinctrl-0 = <&button_pins>;
+		pinctrl-names = "default";
+
+		button@1 {
```

Comment:
`button-reset` (`gpio-keys` doesn't use `reg` property and `@` shoudn't be used)